src/pages/homePage.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
Please rename this Module to `ForgotPassword.js` or `ForgotPassword/index.js`
Pages and Components should start with capital letters